README.md

```diff
@@ -214,6 +214,8 @@ requirements. See table below.
 | Plural module | cldr/supplemental/plurals.json (for cardinals)<br>cldr/supplemental/ordinals.json (for ordinals) |
 | Relative time module | cldr/main/`locale`/dateFields.json<br>+CLDR JSON files from number and plural modules |
 
+As alternative to deducing this yourself you could try using a tool called [Globalize · So What'cha Want](http://johnnyreilly.github.io/globalize-so-what-cha-want/).  This allows you to select the modules you're interested in using and tell you the Globalize files *and* CLDR JSON that you need. 
```

Comment:
:+1: to @sompylasar and @rxaviers comments. 

Also:
* Change "As alternative" to "As an alternative"
* comma after "yourself"
* Change "you could try using" to  "you could use"

So the first sentence would look like this:

> As an alternative to deducing this yourself, you could use a tool called \[Globalize · So What'cha Want](http://johnnyreilly.github.io/globalize-so-what-cha-want/).